py4DSTEM/process/fit/fit.py

```diff
@@ -24,20 +25,27 @@ def fit_1D_gaussian(xdata,ydata,xmin,xmax):
     A,mu,sigma = scale*popt[0],popt[1],popt[2]
     return A,mu,sigma
 
-def fit_2D(function, data, data_mask=None, popt_guess=None):
+def fit_2D(function, data, data_mask=None, popt_guess=None, 
+    robust=False, robust_steps=3, robust_thresh=2):
     """
     Performs a 2D fit, where the fit function takes its first input in the form of a length 2
     vector (ndarray) of (x,y) positions, followed by the remaining parameters, and the data to
-    fit takes the form of an (n,m) shaped array.
```

Comment:
I think maybe this line was deleted by accident?

Reply:
Yes it was!  Fixed.